com.github.dynobo.normcap.yml

```diff
@@ -0,0 +1,94 @@
+app-id: com.github.dynobo.normcap
+runtime: org.kde.Platform
+runtime-version: 6.2
+sdk: org.kde.Sdk
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --device=dri
+  - --env=TESSDATA_PREFIX=/app/share
+command: normcap
+cleanup:
+  - /sbin
+  - /include
+  - /lib/cmake
+  - /lib/pkgconfig
+  - /bin/protoc
+  - /share/doc
+  - /share/pkgconfig
+  - /share/man
+  - /share/cmake*
+  - "*.a"
+  - "*.la"
+
+modules:
+  - name: kerberos
+    buildsystem: autotools
+    subdir: src
+    sources:
+      - type: archive
+        url: https://kerberos.org/dist/krb5/1.20/krb5-1.20.tar.gz
+        sha256: 7e022bdd3c851830173f9faaa006a230a0e0fdad4c953e85bff4bf0da036e12f
+      - type: shell
+        commands:
+          - rm -r .github doc NOTICE README
```

Comment:
why do you still need this ?

Reply:
True! Not needed anymore with `subdir`...